guides/basic-use/deploying.md

```diff
@@ -96,33 +96,51 @@ Lastly, if you find yourself working with an older app, you may see references t
 
 ### Content Security Policy
 
-It is a best practice to define a Content Security Policy for your application, even for local development, and to make it restrictive. That way security violations can be discovered immediately.
+It is a best practice to define a Content Security Policy (CSP) for your application and to make it restrictive. A Content Security Policy configuration defines the list of places that your app should accept data from. For example, an app may allow stylesheets from a CDN and images from a specific data storage service. By forbidding everything else common Cross-Site-Scripting (XSS) attacks could be prevented.
 
-A Content Security Policy configuration defines the list of places that your app should accept data from. For example, an app may allow stylesheets from a CDN and images from a specific data storage service.
+The [`ember-cli-content-security-policy`](https://github.com/rwjblue/ember-cli-content-security-policy) addon helps you to develop and test your application with a CSP applied. It could also inject the CSP in the app as a meta tag on build time. As any other addon it should be installed using the Ember CLI:
```

Comment:
Are there other addons similar to this?

Reply:
Not that I am aware of.

Reply:
I think it depends on your criteria for similarity:

- This addon helps you to develop addons and apps that are compliant with a specific content security policy. This is somewhat similar to `ember-a11y-testing`, which helps you to develop apps and addons that are compliant with specific accessible standards (e.g. WCAG 2.0 Level A).
- It makes usage of a security feature (content security policy) easy for an ember application. This could be seen as similar to `ember-cli-sri`, which also helps to use a security feature (Subresource Integrity) in an ember application.

But there is no other addon that deal with Content Security Policy as far as I'm aware of.